sdk/rust/oak/src/io/mod.rs

```diff
@@ -49,6 +49,20 @@ pub fn node_create<T: Encodable + Decodable>(
     Ok(sender)
 }
 
+/// Creates a node and corresponding inbound channel of the same type, for nodes that are
+/// instantiated via the [`crate::entrypoint_command_handler`] macro.
+pub fn entrypoint_node_create<T: crate::WasmEntrypoint + crate::CommandHandler>(
+    name: &str,
+    label: &Label,
+    wasm_module_name: &str,
```

Comment:
`entrypoint_node_create` only works for entrypoints defined in the same module, right? It seems a bit silly that devs have to specify the current module name manually.

Could we expose a global like `oak::MODULE_NAME` that contains the name of the current module somehow? I suppose we would need to read that from the app manifest?